client/src/components/ShareBtn.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { ShareIcon as ShareBtnSolid } from '@heroicons/react/24/solid'
```

Comment:
No need to `import as` here as there is only one `ShareIcon` that we are importing (the icons imported in other button components had both a Solid version and an Outline version)